Formula/quicktree.rb

```diff
@@ -13,6 +13,8 @@ class Quicktree < Formula
   end
 
   def install
+    # https://github.com/khowe/quicktree/issues/8
+    inreplace "src/distancemat.c", "strlen(identifier)", "(strlen(identifier)+1)"
```

Comment:
I prefer to submit a pull request, and then add a `patch do` that pulls the patch from that pull request. For such a simple fix, this `inreplace` is fine too.